packages/shorebird_cli/lib/src/shorebird_process.dart

```diff
@@ -34,9 +48,6 @@ class ShorebirdProcess {
     Map<String, String>? environment,
     String? workingDirectory,
   }) async {
-    logger.detail(
```

Comment:
this was a superfluous log since this method just proxies to the underlying start call which also logs as detail